src/App.jsx

```diff
@@ -1,28 +1,91 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+
 class App extends React.Component {
   state = {
     todos: [],
+    prepearedTodos: [],
```

Comment:
you could store only one array in state(imagine situation when this array consists of thousands of items)